test/unittests/mongoBackend/mongoSubscribeContext_test.cpp

```diff
@@ -331,7 +330,7 @@ TEST(mongoSubscribeContext, Ent1_Attr0_C1)
     EXPECT_EQ(id, sub.getField("_id").OID().toString());
     EXPECT_EQ(1360236300, sub.getIntField("expiration"));
     EXPECT_FALSE(sub.hasField("lastNotification"));
-    EXPECT_FALSE(sub.hasField("throttling"));
+    EXPECT_EQ(-1, sub.getIntField("throttling"));
```

Comment:
Slight change in how throttling is persisted, but I doesn't involve a functional e2e change at the end. What do you think?